### ISSUE TITLE
TP: Can't assign DS to servers by CDN/Profiles

### ISSUE THREAD

**Issue by smalenfant** [apache/incubator-trafficcontrol#1724]
This might be a "feature parity" one. With Traffic Ops, we could associate DS to servers using profiles before. Doesn't seem possible with new UI.

Workaround is probably using the filters and use the select all button.

**Comment by mitchell852** [comment]
Yep, you got it. You have to filter servers by profile name and then select all

**Comment by dneuman64** [comment]
+1 to Steve's issue.  I acutally like the hierarchy in the old UI where it was by profile -> cachegroup -> individual caches

**Comment by rob05c** [comment]
On the fence for `bug`/`enhancement`, going with feature parity => bug. Feel free to switch.

**Comment by rawlinp** [comment]
This issue may be obsoleted by topology assignment assuming we get around to deprecating legacy ds-server assignments.
